examples/v2/config_chat2.nim

```diff
@@ -190,7 +190,7 @@ type
 
     contentTopic* {.
       desc: "Content topic for chat messages."
-      defaultValue: "/waku/2/huilong/proto"
+      defaultValue: "/toychat/2/huilong/proto"
```

Comment:
```suggestion
      defaultValue: "/toy-chat/2/huilong/proto"
```

"toychat" is not a word. Do we care?

Reply:
No strong feelings about this, but would agree that `toy-chat` is slightly preferred - as long as we now ensure consistency in https://github.com/status-im/js-waku/pull/188 and https://github.com/vacp2p/rfc/tree/master/content/docs/rfcs/23. @oskarth any objections?

Reply:
yakshaving? doesn't matter

Reply:
It is bikeshedding. Up to you @jm-clius.